lib/foreman_openscap/engine.rb

```diff
@@ -32,41 +32,50 @@ class Engine < ::Rails::Engine
                                        'foreman_openscap/policy.css']}}
     end
 
-    initializer 'foreman_openscap.register_plugin', :after=> :finisher_hook do |app|
+    initializer 'foreman_openscap.register_plugin', :after => :finisher_hook do |app|
       Foreman::Plugin.register :foreman_openscap do
         requires_foreman '>= 1.5'
 
         # Add permissions
         security_block :foreman_openscap do
-          permission :view_arf_reports, {:scaptimony_arf_reports => [:index, :show],
-                                         :scaptimony_policies => [:index, :show],
+          permission :view_arf_reports, {:scaptimony_arf_reports   => [:index, :show, :parse, :auto_complete_search],
+                                         :scaptimony_policies      => [:index, :show],
+                                         :scaptimony_dashboard     => [:index],
+                                         :scaptimony_hosts         => [:show],
                                          :scaptimony_scap_contents => [:index, :show],
-                                        }
-          permission :edit_compliance, {:scaptimony_arf_reports => [:destroy],
-                                        :scaptimony_policies => [:new, :create, :edit, :update, :destroy,
-                                          :select_multiple_hosts, :upldate_multiple_hosts],
-                                        :scaptimony_scap_contents => [:new, :create, :edit, :update]
-                                       }
+                                      }
+          permission :edit_compliance, {:scaptimony_arf_reports           => [:destroy],
```

Comment:
don't we miss view_compliance that should combine scaptimony_policies => index, show, auto_complete_search?